src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/Example5.properties

```diff
@@ -0,0 +1,5 @@
+key.one=41 # // SUPPRESS CHECKSTYLE because I want to
+key.one=42 # ok as within line range
+key.one=43 # ok as within line range
+key.two=44 # // violation 'Duplicated property 'key.two' (2 occurrence(s)).'
+key.two=45
```

Comment:
Updated so the violation occurs on a different duplicated key (which doesn't have any suppression). imo easier to understand